File: src/Entity/Reclamation.php
<?php

namespace App\Entity;

use App\Repository\ReclamationRepository;
use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: ReclamationRepository::class)]
class Reclamation
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
    private ?\DateTimeInterface $date = null;

    #[ORM\Column(length: 255)]
    private ?string $email = null;

    #[ORM\Column(type: Types::BIGINT)]
    private ?string $telephone = null;

    #[ORM\Column(type: Types::TEXT)]
    private ?string $cmnt = null;

    #[ORM\Column(length: 255)]
    private ?string $etat = null;

    #[ORM\ManyToOne(inversedBy: 'reclamations')]
    #[ORM\JoinColumn(nullable: true)]
    private ?user $id_user = null;

    #[ORM\ManyToOne(inversedBy: 'reclamations')]
    #[ORM\JoinColumn(nullable: false)]
    private ?TypeReclamation $id_tr = null;

    #[ORM\ManyToOne(inversedBy: 'reclamations')]
    private ?doctor $id_doctor = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getTelephone(): ?string
    {
        return $this->telephone;
    }

    public function setTelephone(string $telephone): self
    {
        $this->telephone = $telephone;

        return $this;
    }

    public function getCmnt(): ?string
    {
        return $this->cmnt;
    }

    public function setCmnt(string $cmnt): self
    {
        $this->cmnt = $cmnt;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }

    public function getIdUser(): ?user
    {
        return $this->id_user;
    }

    public function setIdUser(?user $id_user): self
    {
        $this->id_user = $id_user;

        return $this;
    }

    public function getIdTr(): ?TypeReclamation
    {
        return $this->id_tr;
    }

    public function setIdTr(?TypeReclamation $id_tr): self
    {
        $this->id_tr = $id_tr;

        return $this;
    }

    public function getIdDoctor(): ?doctor
    {
        return $this->id_doctor;
    }

    public function setIdDoctor(?doctor $id_doctor): self
    {
        $this->id_doctor = $id_doctor;

        return $this;
    }
}

File: PiDesktop/src/Entities/reclamation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;

/**
 *
 * @author EMNA
 */
public class reclamation {
     private int id;
    private int id_usr_id;
    private type_reclamation id_tr_id;
    private Date date;
    private String email;
    private int telephone;
    private String cmnt;
    private String etat;
    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId_usr_id() {
        return id_usr_id;
    }

    public void setId_usr_id(int id_usr_id) {
        this.id_usr_id = id_usr_id;
    }

    public type_reclamation getId_tr_id() {
        return id_tr_id;
    }

    public void setId_tr_id(type_reclamation id_tr_id) {
        this.id_tr_id = id_tr_id;
    }
    
    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getTelephone() {
        return telephone;
    }

    public void setTelephone(int telephone) {
        this.telephone = telephone;
    }

    public String getCmnt() {
        return cmnt;
    }

    public void setCmnt(String cmnt) {
        this.cmnt = cmnt;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }


    public reclamation(int id, int id_usr_id, type_reclamation id_tr_id, Date date, String email, int telephone, String cmnt, String etat) {
        this.id = id;
        this.id_usr_id = id_usr_id;
        this.id_tr_id = id_tr_id;
        this.date = date;
        this.email = email;
        this.telephone = telephone;
        this.cmnt = cmnt;
        this.etat = etat;
    }
    public reclamation(){
    }

    public reclamation(int id, int id_usr_id, Date date, String email, int telephone, String cmnt, String etat) {
        this.id = id;
        this.id_usr_id = id_usr_id;
        this.date = date;
        this.email = email;
        this.telephone = telephone;
        this.cmnt = cmnt;
        this.etat = etat;
    }

    public reclamation(int id_usr_id, type_reclamation id_tr_id, Date date, String email, int telephone, String cmnt, String etat) {
        this.id_usr_id = id_usr_id;
        this.id_tr_id = id_tr_id;
        this.date = date;
        this.email = email;
        this.telephone = telephone;
        this.cmnt = cmnt;
        this.etat = etat;
    }

    @Override
    public String toString() {
        return "reclamation{" + "id=" + id + ", id_usr_id=" + id_usr_id + ", type_reclamation=" + id_tr_id + ", date=" + date + ", email=" + email + ", telephone=" + telephone + ", cmnt=" + cmnt + ", etat=" + etat + "\n";
    }


    
}
